xlators/protocol/server/src/server.c

```diff
@@ -1867,9 +1870,13 @@ struct volume_options server_options[] = {
      .type = GF_OPTION_TYPE_SIZET,
      .min = GF_MIN_SOCKET_WINDOW_SIZE,
      .max = GF_MAX_SOCKET_WINDOW_SIZE,
-     .description = "Specifies the window size for tcp socket.",
+     .default_value = "0",
```

Comment:
same here?

Reply:
No. Zero means to not use `setsockopt()` to set (both send and receive) window size and so just use OS defaults.